### ISSUE TITLE
Fix CPU stats

### ISSUE THREAD

**Issue by bricktrooper** [droid-101/ServerUI#3]
Use `top -b -n 1 -p 2036`.  The script will require the PID of the server.

**Comment by bricktrooper** [comment]
Use `mpstat -P ALL 1 1`.  This will display 1 report all 4 core utilization levels over a period of 1s.  There will also be a summary at the end.  Pick either one.  This is very hard to parse in bash scripts so make a `python3` script that runs this and adds up the CPU percentages.  Then call this python script from `system-stats.sh`.

**Comment by bricktrooper** [comment]
Honestly just convert `system-stats.sh` to a python3 script

**Comment by droid-101** [comment]
done
